### ISSUE TITLE
Object entry error

### ISSUE THREAD

**Issue by dockramer** [bschlenk/homebridge-roku#6]
I've got 3 roku sticks that are correctly identified by roku-config and added to config.json, but I'm unable to start homebridge  and I get this error...

[12/28/2017, 4:06:05 PM] [Roku] Initializing Roku accessory...

/usr/local/lib/node_modules/homebridge-roku/src/homebridge-roku.js:116
    return Object.entries(this.appMap)
                  ^
TypeError: Object.entries is not a function
    at RokuAccessory.setupChannels (/usr/local/lib/node_modules/homebridge-roku/src/homebridge-roku.js:116:19)
    at RokuAccessory.setup (/usr/local/lib/node_modules/homebridge-roku/src/homebridge-roku.js:33:32)
    at new RokuAccessory (/usr/local/lib/node_modules/homebridge-roku/src/homebridge-roku.js:24:10)
    at Server._loadAccessories (/usr/local/lib/node_modules/homebridge/lib/server.js:275:29)
    at Server.run (/usr/local/lib/node_modules/homebridge/lib/server.js:86:38)
    at module.exports (/usr/local/lib/node_modules/homebridge/lib/cli.js:40:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:456:32)
    at tryModuleLoad (module.js:415:12)
    at Function.Module._load (module.js:407:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:159:18)

Thoughts?
Thanks

**Comment by bschlenk** [comment]
You must be using an older version of node. I meant to support node 6 or higher, I'll make some changes and remove Object.entries, it wasn't added until some time after node 6.

**Comment by dockramer** [comment]
node -v returns 6.0.0

thank you for your quick resposnse!

**Comment by bschlenk** [comment]
Can you try updating homebridge-roku and running it again? I just published an update.

**Comment by dockramer** [comment]
Got a UUID naming error.

changed each entry name to it's corresponding name (Bedroom, family room, etc) and homebridge loads all 3 without an error.  have'n't had a chance to see how it works with siri yet.  I'll let you know.

Thanks again

**Comment by bschlenk** [comment]
I’m curious how you have it set up. homebridge-roku-config doesn’t recognize multiple devices yet as far as I know, did you have to add them manually?

**Comment by dockramer** [comment]
No, it added them automatically.  I think I ran config several times and they were all eventually added.  All I had to do was change the name for each of them.  But on homekit, i've got a ton of switches now for each app, volumeup, voluomedown, volumeup, poweron.

Each switch works turning the selected app on, but it won't turn the TV on.
Volume switches don't do anything
Poweron switch doesn't do anything

Would be nice if switching on app would start it up, and turning switch off would go home.
